licensing/templates/main-menu.php

```diff
@@ -0,0 +1,159 @@
+
+<?php
+/**
+ * Module-specific PHP for creating the main menu links in #main-menu-titles
+ * Included in ../templates/title.php
+ */
+
+
+if ($user->isAdmin()) { ?>
+
+    <a href='index.php'>
+        <div class="main-menu-link <?php if ($currentPage == 'index.php') { echo "active"; } ?>">
```

Comment:
A `<div>` element inside an `<a>` element is invalid HTML 4.01, which is what CORAL currently outputs. These should all be switched around.